front/components/vaults/VaultAppsList.tsx

```diff
@@ -105,12 +119,23 @@ export const VaultAppsList = ({ owner, onSelect }: VaultAppListProps) => {
             setAppSearch(s);
           }}
         />
+        {isBuilder && (
+          <Button
+            label="New App"
+            variant="primary"
+            icon={PlusIcon}
+            size="sm"
+            onClick={() => {
+              setIsCreateAppModalOpened(true);
+            }}
+          />
+        )}
       </div>
       <DataTable
         data={rows}
         columns={getTableColumns()}
         filter={appSearch}
-        filterColumn="label"
+        filterColumn="Name"
```

Comment:
Here: Fixing the search feature I broke on my last PR when renaming the column.

Reply:
🙏 ideally column name must be lowercased, only the header should be uppercase.